packages/expo-updates/android/src/androidTest/java/expo/modules/updates/UpdatesUtilsInstrumentationTest.kt

```diff
@@ -18,7 +18,7 @@ class UpdatesUtilsInstrumentationTest {
     )
   }
 
-  @Test
+  @Test(expected = ParseException::class)
```

Comment:
it's unclear which test Assert should fail. prefer to separate these out into ones that are expected to fail and ones that aren't.

Also could be a problem that these succeeded in the past but no longer succeed.

Reply:
These tests were always throwing `ParseException` so this should have been there already, maybe the test framework is more strict now.

Reply:
These don't seem to throw at 023541a658 (the commit above your large one).

Reply:
Ok I'll have a look. Also I'm seeing a regression where some of the Android updates E2E tests are failing, does not seem to be due to the changes in this PR. Investigating.

Reply:
Ok the regression was due to my change to `UpdatesUtils.kt`, PR is updated and all CI should now pass.

Reply:
Cool! Back to my original comment: 

> it's unclear which test Assert should fail. prefer to separate these out into ones that are expected to fail and ones that aren't.

This test has two assert cases but only the first one runs because it throws. While the test succeeds, the second case doesn't run, so we lost test coverage. Either separate out the two cases into two tests so we can assert that they both throw, or, if it's no longer useful to have the second assert case we can just remove it.

Reply:
Done